libs/client/use-cases/dashboard/shared/ui/src/lib/footer/footer.component.html

```diff
@@ -1 +1,6 @@
-<p>footer works!</p>
+<div>
+        <p class="tui-text_body-m">"Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
```

Comment:
if the storybook doesn't currently support taiga styles then please don't use it here.